### PR TITLE
Links How to contribute to Flutter YouTube video

### PR DESCRIPTION
**Pull request by werainkhatri**
Links to the famous `How to contribute to Flutter` video from the YouTube channel in the CONTRIBUTING.md doc.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ presented.
    practice to do before coding more intricate changes.
 
 In addition to the above, there are many pages on [our
-Wiki](https://github.com/flutter/flutter/wiki/) and an article [Contributing to Flutter:
+Wiki](https://github.com/flutter/flutter/wiki/), a video [How to contribute to Flutter](https://www.youtube.com/watch?v=4yBgOBAOx_A) on the [Flutter](https://www.youtube.com/c/flutterdev/) YouTube channel and an article [Contributing to Flutter:
```
Comment (Hixie) [inline CONTRIBUTING.md:101]: need a comma after "channel"

Comment (Hixie) [reply CONTRIBUTING.md:101]: should we just inline the video?

Comment (werainkhatri) [reply CONTRIBUTING.md:101]: Yeah, that sounds better.

Comment (werainkhatri) [reply CONTRIBUTING.md:101]: My next question would be where?

Comment (werainkhatri) [reply CONTRIBUTING.md:101]: GitHub Markdown does not support inlining the video ([link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos)). There is an alternative, which is not bad.

**Comment by Hixie** [comment]
LGTM

**Review by Hixie — APPROVED** [review]
LGTM

**Pull request merged**